libraries/nng/nng.SlackBuild

```diff
@@ -25,10 +25,11 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=nng
-VERSION=${VERSION:-1.8.0}
+VERSION=${VERSION:-1.9.0}
 BUILD=${BUILD:-1}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
+NUMJOBS=${NUMJOBS:-" -j$(expr $(nproc) + 1) "}
```

Comment:
no - parse out of makeflags instead

Reply:
An example in the repo:

https://github.com/SlackBuildsOrg/slackbuilds/blob/3e6f0582a860188912565acd98f8fc6cbd979b23/development/godot/godot.SlackBuild#L84-L89